common/components/controllers/PrivacyController.jsx

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import React from "react";
+
+class PrivacyController extends React.PureComponent<{||}> {
+  render(): $React$Node {
+    return (
+      <div className="container privacy-root">
+        <div className="row">
+          <div className="col-12">
+            <h1>Privacy Policy</h1>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+export default PrivacyController;
```

Comment:
Example of a basic front end page template demonstrating the Bootstrap container/row/column classes used by DLab and no page content beyond the title.